core/comp_ui.py

```diff
@@ -189,22 +193,23 @@ def _PrintCandidates(self, unused_subst, matches, unused_match_len):
 
     self._RedrawPrompt()
 
-  def PrintRequired(self, msg, *args):
-    # type: (str, *Any) -> None
-    self.f.write('\n')
-    if args:
-      msg = msg % args
-    self.f.write(' %s\n' % msg)  # need a newline
-    self._RedrawPrompt()
+  if mylib.PYTHON:
+    def PrintRequired(self, msg, *args):
+      # type: (str, *Any) -> None
+      self.f.write('\n')
+      if args:
+        msg = msg % args
+      self.f.write(' %s\n' % msg)  # need a newline
+      self._RedrawPrompt()
 
 
 def _PrintPacked(matches, max_match_len, term_width, max_lines, f):
-  # type: (List[str], int, int, int, IO[bytes]) -> int
+  # type: (List[str], int, int, int, mylib.Writer) -> int
   # With of each candidate.  2 spaces between each.
   w = max_match_len + 2
 
   # Number of candidates per line.  Don't print in first or last column.
-  num_per_line = max(1, (term_width-2) // w)
+  num_per_line = max(1, int((term_width-2) / w))
```

Comment:
Hm this might be equivalent, but I think it's also a good idea to translate `//` to `/` in mycpp

Let me see if I can add that